src/dataServices/dynamoDbUtil.test.ts

```diff
@@ -50,6 +51,8 @@ describe('cleanItem', () => {
 describe('prepItemForDdbInsert', () => {
     const id = '8cafa46d-08b4-4ee4-b51b-803e20ae8126';
     const vid = 1;
+    const organization = 'Organization/1';
+    const otherPatient = 'Patient/pat2';

```

Comment:
Ah, let me move it inside the test where it's being used so the context is clearer and it's easier to read. The reason I declared these two strings as variables is because they're each used twice. Once as part of the BUILD step in "BUILD, OPERATE, CHECK", and the second time as part of the CHECK step. It's important that the values are the same in both steps.